### PR TITLE
use Project.toml + clean up test deps

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

**Comment by mkborregaard** [comment]
Yay :heart:
do we want to register with TagBot?

**Comment by SimonDanisch** [comment]
If it passes, we should use Registrator & Tagbot ;)

**Comment by daschw** [comment]
I think we should register PlotReferenceImages.

**Comment by SimonDanisch** [comment]
https://github.com/JuliaPlots/PlotReferenceImages.jl/issues/40

File: test/imgcomp.jl [hunk 1]
```diff
@@ -31,6 +12,33 @@ default(size=(500,300))
...
+        else
+            rm(tarfile, force = true)
+        end
+    else
+        @info "using reference images for version $version (already downloaded)"
+    end
+    refpath
+end
+
+const ref_image_dir = download_reference()
```
Comment (daschw) [inline test/imgcomp.jl]: With theses changes we should probably also change http://docs.juliaplots.org/latest/contributing/#visualregressiontests, which should be updated anyway.
I'm just commenting, not saying that you should do this @SimonDanisch, thanks for your efforts!

**Comment by SimonDanisch** [comment]
Yay it's green :) Should we merge? Maybe squash before, I guess^^

**Pull request merged**